imap_processing/spice/pointing_frame.py

```diff
@@ -4,17 +4,29 @@
 import typing
 from collections.abc import Generator
 from contextlib import contextmanager
+from datetime import datetime, timezone
 from pathlib import Path
 
 import numpy as np
 import spiceypy
 from numpy.typing import NDArray
 
+from imap_processing.spice.geometry import SpiceFrame
 from imap_processing.spice.kernels import ensure_spice
-from imap_processing.spice.time import met_to_sclkticks, sct_to_et
+from imap_processing.spice.repoint import get_repoint_data
+from imap_processing.spice.time import et_to_utc, met_to_sclkticks, sct_to_et
 
 logger = logging.getLogger(__name__)
 
+POINTING_SEGMENT_DTYPE = np.dtype(
+    [
+        ("sclk_start", np.float64),
+        ("sclk_end", np.float64),
+        ("quaternion", np.float64, (4,)),
+        ("pointing_id", np.int32),
```

Comment:
Neither of these may matter, but figured I'd mention them in case.

Should sclk be a uint? I thought MET was the float version.

Should pointing_id be uint?